crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -547,11 +547,15 @@ impl FromWorld for BloomPipelines {
                     format: TextureFormat::Rg11b10Float,
                     blend: Some(BlendState {
                         color: BlendComponent {
-                            src_factor: BlendFactor::One,
+                            src_factor: BlendFactor::SrcAlpha,
```

Comment:
The blend factors need an explanation both here, and in the shader, about why they're used like this.

Reply:
Will add that. Should i explain why we are using alpha blending or that setup is alpha blending?

Reply:
Explain how the alpha doesn't actually matter because our texture is only RGB (not RGBA), but we use it to lerp between the src and dest textures.